src/recommendations/src/recommendations-service/experimentation/experiment_optimizely.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from optimizely import optimizely
+
+from . import experiment, resolvers
+
+
+optimizely_sdk = optimizely.Optimizely(sdk_key=os.environ('OPTIMIZELY_SDK_KEY'))
+
+
+class OptimizelyFeatureTest(experiment.Experiment):
+    def get_items(self, user_id, current_item_id = None, item_list = None, num_results = 10, tracker = None):
+        assert user_id, "`user_id` is required"
+
+        # All the kwargs that are passed to ResolverFactory.get will be stored as a JSON feature variable.
+        resolver_init_kwargs = optimizely_sdk.get_feature_variable_json(self.feature, 'resolver_init_kwargs',
+                                                                        user_id=user_id)
```

Comment:
What do we get back here? Will you add some documentation to help the reader understand what Optimizely is returning and how it's used in the following call?

Reply:
Agreed to document more fully in subsequent PR.